test/test_browser.py

```diff
@@ -5507,8 +5507,13 @@ def test_webpack(self, es6):
     shutil.copy('webpack/src/hello.wasm', 'webpack/dist/')
     self.run_browser('webpack/dist/index.html', '/report_result?exit:0')
 
-  def test_vite(self):
+  @parameterized({
+    '': ([],),
+    'pthreads': (['-pthread', '-sPTHREAD_POOL_SIZE=1', '-sENVIRONMENT=web,worker'],),
```

Comment:
We need to compile for only browser based environments, otherwise Vite runs into issues with bundling.

Perhaps it could be worth moving `-sEnvironment` to the default args array below.